test/cim/iec61968/assetinfo/test_power_transformer_info.py

```diff
@@ -0,0 +1,21 @@
+#  Copyright 2021 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from test.cim.constructor_validation import ai_kwargs, pti_args, verify_power_transformer_info_constructor, verify_pti_args
+from zepben.evolve import PowerTransformerInfo, TransformerTankInfo
+
+
+def test_power_transformer_info_constructor_kwargs():
```

Comment:
this needs to be like `test_pec_constructor_kwargs`, with a decorator of `@given(pti=**pti_kwargs)`, which will make it a hypothesis property-based test.